lua/tardis/languages/en.lua

```diff
@@ -122,6 +123,7 @@ T.Phrases = {
     ["Controls.EngineRelease"] = "Engine Release",
     ["Controls.EngineRelease.FailedDemat"] = "Failed to dematerialise",
     ["Controls.EngineRelease.ForceDemat"] = "Force dematerialisation triggered",
+    ["Controls.EngineRelease.ArtronAdded"] = "Health has been wasted to charge artron energy in the vortex",
```

Comment:
I think it would be better to say 'sacrificed' rather than 'wasted', also need to consider what this says when health is not enabled as in the other comment

Reply:
e27baa3093c9153aef40eefea879a608d98ae0bd